organizational-structure-overview.md

```diff
@@ -13,15 +13,26 @@ The OpenSSF is comprised of instances of the following categories of official gr
-   - A **Technical Deliverable** is an open source licensed software (source code) and its supporting artifacts (technical documentation, etc.) produced from a project. 
-- A **Working Group** (WG) is a group (or team) based Technical Initiative formed to support the overall mission of the OpenSSF. WGs generally do not produce open source software as a primary artifact. WGs often include some open source code, or use licensed software, in fulfillment of their Charter. WGs can be formed to perform tasks Temporary or Ongoing in scope.
-  - **Temporary** scope -- The WGs charter may specify that the WG will wrap up after a specific event, such as a publication.
-  - **Ongoing** scope -- The WGs charter may specify that an effort is expected to be ongoing in support of open source security indefinitely.
-- A **Service** is a Technical Initiative in which software is either built or acquired to support or automate OSSF transactions
-  
+- **Technical Initiatives** are groups formed in support of the overall mission of the OpenSSF. Technical Initiatives include Working Groups (WG), Special Interest Groups (SIG), Projects, and services governed by their own Charters with various objectives, reporting relationships, and funding models described below.
+   - All Technical Initiatives are open groups focused on a technical objective, with transparent proceedings that are open to the public, and vary in their objectives.
+   - Technical Initiatives usually manage one or more GitHub repos of their own, may have separate meetings.
+- A **Project** is a Technical Initiative focused on the development and ongoing support of open source licensed software (source code) and its supporting artifacts (technical documentation, etc.).
+- A **Working Group** (WG) focuses on developing other types of deliverables than software such as guides, specifications, and educational material, or conducting initiatives such as an education outreach. While a WG does not produce open source software as a primary artifact it may launch Projects that do so. A WG may also launch Special Interest Groups (SIG) to perform specific tasks. WGs often include some open source code, or use licensed software, in fulfilment of their Charter.
+- **Special Interest Groups** (SIG) are under the direct governance of their reporting WGs and are bound to achieving a very specific goal. 
+- A **Technical Deliverable** can be an open source licensed software and its supporting artifacts, a specification, a technical guide. This is a technical artefact produced by a Technical Initiative.
+- A **Service** is a Technical Initiative in which software is either built or acquired to support or automate OSSF transactions.
+
+The following table describes the main types of groups and their characteristics.
+
+| Initiative | Expected lifespan | Primary output| Reporting relationship |	Economic model
```

Comment:
"Reporting Relationship" seems to answer the question, "What body approves the creation of $Initiative?" but line 31 gives two options. I think we could clarify this by either: adding an "Approving body" column or clarifying in line 19 who approves what kinds of projects (distinguishing between the tooling a WG needs to create vs a Project that is intended to be adopted outside of the WG)

Reply:
The concept of an approving body is (IMO) a good idea regardless as it provides a concentrated point of review, support, or even correction.  Reporting relationships don't necessarily imply an approval and it may be worth while to call out so that we can detail which of said items do or do not require the approval.

Reply:
I think that's a valid point but that is better addressed in a followup PR.

Reply:
does that PR exist?  We don't want to lose this as an action item. even creating a stub in the table with "approving body == TBD" will be enough.